backend/core/src/seeds/listings/listing-default-bmr-chart-seed.ts

```diff
@@ -1,18 +1,47 @@
 import { ListingDefaultSeed } from "./listing-default-seed"
-import { getDefaultUnits } from "./shared"
+import { getDefaultUnits, getDefaultProperty, getDefaultAmiChart } from "./shared"
```

Comment:
This seed listing just wasn't working anymore and we must have missed it - so now I'm actually adding the appropriate units so we get the BMR chart